.travis.yml

```diff
@@ -22,13 +22,18 @@ matrix:
       compiler: gcc
       env: GMOCK_VER=1.8.0 VALGRIND_TESTS=ON
 
+before_script:
+  - git clone https://github.com/cgreen-devs/cgreen.git
```

Comment:
Given that after several months there is still no sign of a 1.1.0 release, I think we should fix our build to a specific git hash. I want to avoid our build failing when someone makes incompatible changes to Cgreen.